source/templates/pmc-member-results.txt

```diff
@@ -0,0 +1,16 @@
+To: private@[PROJECT].apache.org
+Subject: [RESULT] [VOTE] New PMC member: [Candidate Name]
+
+The vote has now closed. The results are:
+
+Binding Votes:
+
```

Comment:
```suggestion
```
If each line mentions `binding` and `non-binding` then I don't think this is just a list of "Binding Votes:".  Also, it is a bit odd to have two colon-terminated statements in a row.